lib/filesystem/FileStream.cpp

```diff
@@ -91,37 +91,34 @@ static voidpf ZCALLBACK fopen64_file_func(voidpf opaque, const void* filename, i
 			if (mode & ZLIB_FILEFUNC_MODE_CREATE)
 				mode_fopen = "wb";
 
-	if ((filename != NULL) && (mode_fopen != NULL))
-		file = FOPEN_FUNC((const char*)filename, mode_fopen);
+	if((filename != nullptr) && (mode_fopen != nullptr))
+		file = FOPEN_FUNC((const char *)filename, mode_fopen);
```

Comment:
No static_cast here?

Reply:
This code is vanished from develop, so, no need for modernize.